### PR TITLE
[W-11027231] - Recurring Donation Permission Set

### PR DESCRIPTION
**Pull request by JoseRossiSF**
# Work Item
[W-11027231](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00000vPmFzYAK/view)

# Changes
Adding Permission Set for External Partner Users working with Recurring Donations.

### CODE REVIEW

**Review by gerardorodriguezsf — APPROVED** [review]
LGTM

**Review by gerardorodriguezsf — CHANGES_REQUESTED** [review]
@JoseRossiSF 
- npe03__Recurring_Donation__c.npe03__Organization__c should not be editable.

File: force-app/main/default/permissionsets/Recurring_Donations.permissionset-meta.xml [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        <field>npe03__Recurring_Donation__c.npe03__Recurring_Donation_Campaign__c</field>
+        <readable>true</readable>
+    </fieldPermissions>
+    <fieldPermissions>
+        <editable>false</editable>
+        <field>npe03__Recurring_Donation__c.npe03__Total__c</field>
+        <readable>true</readable>
+    </fieldPermissions>
+    <hasActivationRequired>false</hasActivationRequired>
+    <label>Recurring Donations</label>
```
Comment (lparrott) [inline force-app/main/default/permissionsets/Recurring_Donations.permissionset-meta.xml:124]: Let's discuss a better name for this, I posted in Quip with a suggestion, but we should involve the Docs team.

Comment (JoseRossiSF) [reply force-app/main/default/permissionsets/Recurring_Donations.permissionset-meta.xml:124]: Agreed. I have tagged Melissa Barber to our [Quip Doc discussion](https://salesforce.quip.com/Re41AqjiE7G6)

**Comment by JoseRossiSF** [comment]
> @gerardorodriguezsf 
> 
> * npe03__Recurring_Donation__c.npe03__Organization__c should not be editable.

Hi Gerardo, please note in this case the field mentioned represents the Lookup relationship with an Account. I believe in this case we should leave it editable

**Review by maurimart — APPROVED** [review]
LGTM

**Review by gerardorodriguezsf — APPROVED** [review]
LGTM

**Review by lparrott — APPROVED** [review]
Works well. We may remove CommitmentId__c after that field gets pulled out of the permission check, but this looks good for now.

**Review by bdvorachek — APPROVED** [review]

**Pull request merged**